packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1634,20 +1641,25 @@ class PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
           child: child,
         );
       }
-      return child;
+      return Semantics(expanded: _isMenuExpanded, child: child);
     }
 
-    return IconButton(
-      key: StandardComponentType.moreButton.key,
-      icon: widget.icon ?? Icon(Icons.adaptive.more),
-      padding: widget.padding,
-      splashRadius: widget.splashRadius,
-      iconSize: widget.iconSize ?? popupMenuTheme.iconSize ?? iconTheme.size,
-      color: widget.iconColor ?? popupMenuTheme.iconColor ?? iconTheme.color,
-      tooltip: widget.tooltip ?? MaterialLocalizations.of(context).showMenuTooltip,
-      onPressed: widget.enabled ? showButtonMenu : null,
-      enableFeedback: enableFeedback,
-      style: widget.style,
+    return MergeSemantics(
```

Comment:
If you wrap the icon with semantics(expanded) instead of wraping the entire iconButton, you don't need the mergesemantics I think

Reply:
Done!